README.md

```diff
@@ -11,7 +11,11 @@ You can download the community pack created by InsightOps at [Docker pack](https
 ## Usage as a Container
 
 The simplest way to forward all your container's log to the Rapid7 Insight Platform is to
-run this repository as a container, with:
+run this repository as a container.  
+
+If you are using this container in production please ensure that you pin the version of the image.  
```

Comment:
Would it be clearer to leave the `with:` and move this paragraph to after the code?

Reply:
yeah it would